scripts/globals/abilities/pets/somnolence.lua

```diff
@@ -19,8 +18,10 @@ ability_object.onPetAbility = function(target, pet, skill)
     local duration = 120
 
     dmg = dmg*resist
-    dmg = mobAddBonuses(pet, spell, target, dmg, xi.magic.ele.DARK)
-    dmg = finalMagicAdjustments(pet, target, spell, dmg)
+    dmg = mobAddBonuses(pet, nil, target, dmg, xi.magic.ele.DARK)
```

Comment:
I wonder, since this is `onPetAbility`, shouldn't we be referencing `skill` here instead of spell?  Same for the other summoner skills above and below.